cmd/minishift/cmd/delete.go

```diff
@@ -36,6 +37,9 @@ var deleteCmd = &cobra.Command{
 }
 
 func runDelete(cmd *cobra.Command, args []string) {
+	instancedata.Instance().ReadInstanceData()
```

Comment:
correctness and consistency go over "simplicity". If not as part of the config directory, we could also use the _machines_ directory (not the actual libmachine managed directory _minshift_). We could call it _minishift.json_ or more generic _\<machine-name\>.json_. This solves indirectly the issues that we are still not 100% happy with the name of the config file.